ykrt/pass_finder/cargo_run.py

```diff
@@ -1,22 +1,37 @@
 import os, subprocess, time
 
 def run_test(yk_path,env=None, n=1) :
+    if env is None:
+        env = os.environ.copy()
+    env['PATH'] = f"{yk_path}/bin:{env['PATH']}"
+    env['YK_BUILD_TYPE'] = 'debug'
     prelink_passes = (env or os.environ).get('PRELINK_PASSES', 'Not Set')
     print(f"\033[93mPRELINK_PASSES: {prelink_passes}\033[0m")
-    os.chdir("/home/shreei/research/yk_pv")
-    #subprocess.run(["cargo build"], shell=True, env=env or os.environ)
-    times = []
-    for i in range(n):
-        before = time.time() 
-        c = subprocess.run(["timeout 60 cargo test"], shell=True, env=env or os.environ) 
-        elapsed = time.time() - before
-        times.append(elapsed)
-    mean_time = sum(times) / len(times)
+    os.chdir(yk_path)
+    r = subprocess.run(["timeout 50 cargo test"], shell=True, env=env or os.environ)
```

Comment:
How long does the test suite normally take to run? [I can see that we might need a `timeout` because a bad pass might cause it to get stuck in a loop but I wonder if we need a fairly generous timeout of 2x the normal running time?]

Reply:
Agreed, so the `cargo test` doesn't run for more than 12-15 secs. Changing this to `timeout 20 cargo test`.